test_app.py

```diff
@@ -18,7 +22,9 @@ def test_it_returns_users():
 def test_it_returns_members_by_state():
     global test_browser
     state = 'co'
-    response = test_browser.get(f'/members_by_state/{state}')
+    with vcr.use_cassette('fixtures/vcr_cassettes/members_by_state.yaml'):
+      response = test_browser.get(f'/members_by_state/{state}')
```

Comment:
This is a cool way to user vcr, I wonder if there is a webmock version on this.

Reply:
Hmm I wonder. I was honestly pretty surprised to find vcr for Python. ❤️  VCR!